habitat_sim/robots/mobile_manipulator.py

```diff
@@ -193,9 +196,30 @@ def update(self) -> None:
         # Update the cameras
         for sensor_name in self._head_sensor_names:
             sens_obj = self._sim._sensors[sensor_name]._sensor_object
-            head_T = self._get_head_cam_transform()
+
+            look_at = self.sim_obj.transformation.transform_point(
+                self.params.head_cam_look_pos
+            )
+            cam_pos = self.sim_obj.transformation.transform_point(
+                self.params.head_cam_offset_pos
+            )
+            head_T = mn.Matrix4.look_at(cam_pos, look_at, mn.Vector3(0, -1, 0))
```

Comment:
Beware `look_at` will produce a division-by-zero C++ exception/crash if `look_at - cam_pos` is parallel to `mn.Vector3(0, -1, 0)`.

Reply:
I see, is there a safer method to use? Or should I include a manual check so this doesn't happen?

Reply:
If you're feeling ambitious, you could create a new helper for us that does this near-parallel check and falls back to a different "up" vector. But this is a pretty unlikely crash so maybe not a priority.

Reply:
Yeah, let's address this in another PR if it becomes an issue.